tests/unit/test_req_uninstall.py

```diff
@@ -0,0 +1,49 @@
+import os
+import shutil
+import sys
+import tempfile
+
+import pytest
+from mock import Mock
+
+from pip.locations import running_under_virtualenv
+from pip.req.req_uninstall import UninstallPathSet
+
+
+class TestUninstallPathSet(object):
+    def setup(self):
+        if running_under_virtualenv():
+            # Construct tempdir in sys.prefix, otherwise UninstallPathSet
+            # will reject paths.
+            self.tempdir = tempfile.mkdtemp(prefix=sys.prefix)
```

Comment:
I'd rather not have the pip test suite creating temp in sys.prefix.
how about a monkeypatch for  `is_local` instead